packages/utils/src/crypto.js

```diff
@@ -0,0 +1,7 @@
+import crypto from 'crypto'
+
+export const createMd5HashFromString = (string, length = 20) => {
+  return crypto.createHash('md5').update(string).digest('hex').substr(0, length)
+}
+
+export const createTimeHash = () => createMd5HashFromString(new Date().toISOString())
```

Comment:
from a crypto point of view it would probably be better to prefix the time hash with a nuxt-unique seed. That way the hash is really unique

Reply:
Totally right. Wasn't sure if that's necessary or if we even should have an "app-specific" seed to.